# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clanlib06/clanlib_glxfix.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Sources/GL/GLX/displaycard_glx.cpp	2003-04-28 10:07:51.000000000 +0200
+++ ./Sources/GL/GLX/displaycard_glx.cpp	2003-04-28 10:11:54.000000000 +0200
@@ -18,6 +18,9 @@
 #include <X11/Xutil.h>
 #include <X11/cursorfont.h>
 
+#define GLX_GLXEXT_PROTOTYPES
+#include <GL/glx.h>
+
 #include "displaycard_glx.h"
 #include "GL/Generic/blit_gl_generic.h"
 #include "API/Core/System/cl_assert.h"
